enterprise/frontend/src/metabase-enterprise/content_verification/VerifiedFilter/VerifiedFilter.tsx

```diff
@@ -0,0 +1,9 @@
+import { t } from "ttag";
+import type { SearchFilterComponent } from "metabase/search/types";
+
+export const VerifiedFilter: SearchFilterComponent<"verified"> = {
+  title: t`Verified items only`,
+  type: "toggle",
+  fromUrl: value => value === "true",
+  toUrl: (value: boolean) => (value ? "true" : undefined),
```

Comment:
Can you explain a little about these filter APIs? i.e. how `fromUrl` and `toUrl` work. I figure it might have something to do with hydrating the value from the URL and slapping the value back to the URL when the filter changes.

Reply:
Yeah, I left the descriptions of what they do in the type definition so I don't pollute all of the comments 😄